source/css/style.css

```diff
@@ -1,573 +1,510 @@
+/** 1st Level */
+/** 2nd Level */
+/** 3rd Level */
 html {
-    position: relative;
-    min-height: 100%;
+  position: relative;
+  min-height: 100%;
 }
 
 body {
-    margin-bottom: 60px;
+  margin-bottom: 60px;
 }
-
 body > .container {
-    padding: 0 15px 20px;
-    margin-top: 20px;
+  padding: 0 15px 20px;
+  margin-top: 20px;
 }
 
-h1, h2, h3, h4, h5, h6 {
-    margin-bottom: 1rem;
+h1 {
+  margin-bottom: 1rem;
 }
 
-.search-results {
-    margin-left: 20px;
```

Comment:
Please don't change indentation everywhere according to your editor of preference